zookeeper-client/zookeeper-client-java/src/main/java/org/apache/zookeeper/client/ZKClientConfig.java

```diff
@@ -23,7 +23,7 @@
 import org.apache.yetus.audience.InterfaceAudience;
 import org.apache.zookeeper.ZooKeeper;
 import org.apache.zookeeper.common.ZKConfig;
-import org.apache.zookeeper.server.quorum.QuorumPeerConfig.ConfigException;
+import org.apache.zookeeper.server.quorum.ConfigException;
```

Comment:
This is breaking compatibility, we cannot port this into branch3.5 and it will make transition to 3.6 slower

Reply:
Yes, one of the main reasons (not just this particular instance) that it cannot be separated "nicely".

zookeeper-common and zookeeper-client-java will probably merge back to zookeeper-server.